src/ginis.ts

```diff
@@ -48,23 +57,25 @@ export class Ginis {
   /**
    * Exports functions of the api's with config and url values bound.
    * See documentation of the api for request options.
-   * Inputs are typed objects, outputs unformatted xml.
    */
-  json: {
+  xml: {
```

Comment:
Should this still be nested to point to the underlying structure needed for documentation of XML requests and responses or would it be better of unwrapped?

Reply:
you can unwrap